WooCommerce/Classes/ViewModels/Order Details/OrderDetailsViewModel.swift

```diff
@@ -258,28 +258,21 @@ extension OrderDetailsViewModel {
             let printingInstructionsViewController = ShippingLabelPrintingInstructionsViewController()
             let navigationController = WooNavigationController(rootViewController: printingInstructionsViewController)
             viewController.present(navigationController, animated: true, completion: nil)
-        case .shippingLabelProduct:
-            guard let item = dataSource.shippingLabelOrderItem(at: indexPath), item.productOrVariationID > 0 else {
+        case .shippingLabelProducts:
+            let shippingLabelItems = dataSource.shippingLabelOrderItems(at: indexPath)
+
+            let identifier = AggregatedProductListViewController.classNameWithoutNamespaces
+            guard let productListVC = UIStoryboard.orders.instantiateViewController(identifier: identifier) as? AggregatedProductListViewController else {
+                DDLogError("Error: attempted to instantiate AggregatedProductListViewController. Instantiation failed.")
```

Comment:
I was under the impression that we are moving away from Storyboards. What was the reason to use it for this screen?

Reply:
The reason is that the similar view controller that works with `OrderItem` instead of `AggregatedOrderItem` is in the storyboard. I, therefore, preferred to conform and use the same approach by inserting it in the storyboard.